docs/basic-features/built-in-css-support.md

```diff
@@ -165,7 +165,7 @@ You can use component-level Sass via CSS Modules and the `.module.scss` or `.mod
 Before you can use Next.js' built-in Sass support, be sure to install [`sass`](https://github.com/sass/sass):
 
 ```bash
-npm install sass
+npm install --save sass
```

Comment:
Save has been the default for many major npm versions. We don't need this!

Reply:
@Timer oops, sorry I switched from `npm` to `yarn` right [when it started supporting `--save`](https://github.com/npm/npm/issues/5108#issuecomment-305515940) and completely missed that.

I'll remove `--save` from [here](https://nextjs.org/docs/#setup) then.